### ISSUE TITLE
Yuzu.exe crashing while plaing SMO V1.3

### ISSUE THREAD

**Issue by RomanJ55** [yuzu-emu/yuzu#9829]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

EA 3406

### Description of Issue

Hi there. I played super Mario Odyssey with the most recent patch 1.3 yesterday and the yuzu app crashed twice on me. I have over 200 hours in this game playing on yuzu without any issues whatsoever, But yesterday I decided to do a 100% playthrough. This lead me to new areas and subareas that I haven't seen before. 

First crash was in sand kingdom in the inverted pyramid. There is a 2D section in there. After trying to exit the area through a pipe the yuzu app crashed. Second crash was in the metro kingdom. After playing through the whole story there is a festival event. At the end there is also a large 2D section (or rather many smaller ones back to back). After collecting the multi moon at the very end of the event the yuzu app crashed after the loading screen.

I searched the web for these issues and someone mentioned that it would work if I disable the 1.3 patch for those sections. I tried it and it worked. The yuzu app didnt crash anymore. Basically, disabling the patch 1.3 fixed both of the crashes. I was able to progress the game past those specific points on game version 1.0. 

### Expected Behavior

The yuzu app should not crash

### Reproduction Steps

Play the above described sections with patch 1.3 enabled. 

### Log File

Unfortunatly I dont have the logfile anymore since I restarted the app afterwards. But I have a screenshot of the eventlog from windows: https://i.imgur.com/RBpHBci.png . Its in german but I hope it helps nontheless. 

### System Configuration

CPU: AMD Ryzen 5 1600 AF
GPU/Driver: ASUS R9 270X(R9270X-DC2T-2GD5) / 22.6.1
RAM: 2X 8GB Gskill DDR4 CL16 3200(F4-3200-C16-8GVKB)
OS: Windows 10 21H2(Build 19044.2604)

**Comment by G-Spawn** [comment]
When you're able to post a log that'll help a lot.

**Comment by RomanJ55** [comment]
So i tried to replicate the issues again. I played through the story in sand kingdom up to the point I described above. It didnt crash this time. 

I did update yuzu from 3406 to 3408 and installed some windows updates yesterday. I read on the web that it could also be my Visual C++ Redis being broken or sth. since the file "ucrtbase.dll" is a process of the Microsoft C Runtime Library(?). 

I guess this can be marked as fixed then. I assume the windows update fixed whatever issue i had.
